### PR TITLE
Ports Pet Naming from Goldrat

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

Title
Port and "why it's good for the game" taken from; https://github.com/GalacticStation/GalaxiaStation/pull/65

## Why It's Good For The Game

Much like how loadout items can be renamed, this should probably be a thing too.

Pet owners name their pets a lot. It's just- a thing. We all do it.

## Changelog

:cl: nikothedude
add: Pet owners can now name and describe their pet
/:cl:

### CODE REVIEW

**Review by projectkepler-ru — APPROVED** [review]

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/pet_owner.tsx [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+  description:
+    "If blank, will use the mob's default name. For example, 'axolotl' or 'chinchilla'.",
+  component: FeatureShortTextInput,
+};
+
+export const pet_desc: Feature<string> = {
+  name: 'Pet Description',
+  description: "If blank, will use the mob's default description.",
+  component: FeatureShortTextInput,
+};
```
Comment (theselfish) [inline tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/pet_owner.tsx:25]:
```suggestion
};

```

**Comment by nevimer** [comment]
needs modularity comments as it edits skyrat files

**Comment by theselfish** [comment]
> needs modularity comments as it edits skyrat files

It's a port.

**Comment by theselfish** [comment]
> needs modularity comments as it edits skyrat files

https://github.com/Skyrat-SS13/Skyrat-tg/pull/25848/files
Which is a port of the thing I linked in the OP.

**Comment by theselfish** [comment]
These are both in https://github.com/Bubberstation/Bubberstation/pull/897

**Pull request closed without merging**